websites/Y/YouTube/presence.ts

```diff
@@ -366,7 +366,21 @@ presence.on("UpdateData", async () => {
     ) {
       //Sometimes causes problems
       let user: string;
+      //get user title when viewing a community post
```

Comment:
can you add these comments to the other selectors if you can figure out what they are please 🙏🏻

Reply:
Sure but I don't understand what the 3rd one does

Reply:
that's fine ig, do it for the ones you can

Reply:
Actually nvm